cohort_4/playbook/workshops/portfolio/img/margin-challenge.md

```diff
@@ -1,19 +0,0 @@
-#Challenge: Using margins to adjust elements
```

Comment:
It's a duplicate of `cohort_4/playbook/workshops/portfolio/margin_challenge.md` and had broken links as a result of being in the `img` directory instead of `portfolio`.